### DISCUSSION TITLE
Aiko — Free native app for macOS and iOS

### DISCUSSION THREAD

**Discussion by sindresorhus** [openai/whisper#1300] [Show and tell]
https://sindresorhus.com/aiko

Aiko lets you run Whisper locally on your Mac, iPhone, and iPad. It is powered by [whisper.cpp](https://github.com/ggerganov/whisper.cpp).

Highlights:
- Reader and timestamp view
- Record audio
- Export to text, JSON, CSV, subtitles
- Shortcuts support

The app uses the Whisper large v2 model on macOS and the medium or small model on iOS depending on available memory.

Performance on iOS will increase significantly soon thanks to CoreML support in whisper.cpp.

I'm actively working on more features and improvements. Feedback welcome through the feedback button in the app.

![](https://sindresorhus.com/_astro/screenshot1.2ecc32f3.jpg)
![iOS](https://user-images.githubusercontent.com/170270/235439998-b237ce86-bfc2-4fca-9a9d-692830dd180c.png)

**Comment by bick302** [comment]
@sindresorhus 
Thank you for creating this great app. 😄 I have used Aiko on my iPad Air 5. It's smooth and fast. 
I have a question to ask, Why is it the medium model on iPadOS? According to the Whisper.cpp repository readme, the large model uses ~3.3GB of memory. So can it also be used on the iPad(8G memory)?

**Reply by sindresorhus** [reply]
Aiko bundles the small and medium model, and bundling the large model too would make the app too big. In addition, the large model is simply too slow on an iPad. An even though the iPad has 8 GB memory, doesn't mean iOS will allow the app to use that much.

**Reply by bick302** [reply]
I got it. Thanks for your explanation!

**Reply by Amirgamea** [reply]
Hi there and i wanted to thank you again for this amazing app.

The fact about the app size being too big if it pre-included the large library is totally legit but still having the option as an advanced optional possibility to have access to large library just for some specific language by downloading it like game apps download their data and updates after installation and being able to choose which library it should use would be so good and make the experience something extraordinary even more than it is

**Comment by jasongitmail** [comment]
Would it be possible to use Raycast to enable a global hotkey and have Aiko transcribe the live audio to text in place wherever the cursor is (in any app)?

**Reply by sindresorhus** [reply]
Aiko is too slow for that, since it only uses the large model. Check out [superwhisper](https://superwhisper.com).

**Reply by jasongitmail** [reply]
Thanks. Will take a look

**Comment by SugarRayLua** [comment]
Hi, @sindresorhus. The large model does work on iPad Pro iOS (Hello Transcribe app uses) but is slow.

Fyi :-)

**Comment by sdsnatcher** [comment]
Is it possible to make it work on real time subtitles on iOS? Since I have hearing impairment, I would love this as an accessibility feature.  Apples built-in accessibility feature for this is still beta and leaves **much** to be desired.

**Reply by sindresorhus** [reply]
No, it cannot run when not in the foreground.

**Comment by wingsum93** [comment]
Is it possible to add a database or cache ?
 it it helped a lot if the app can store the previous result. 
Currently the apps only support showing single result at the same time.

**Reply by sindresorhus** [reply]
It's something I plan to do long-term, but not something I have time to do right now.

**Comment by RogerPu** [comment]
亲测可用

**Comment by github-account1111** [comment]
Not free anymore?

**Reply by erickirt** [reply]
I guess not

**Reply by ryanheise** [reply]
MIT is a [commercially friendly license](https://www.revenera.com/software-composition-analysis/glossary/what-is-an-mit-license#commercial-use-of-the-mit-license), provided that the original copyright notice and license are included. I have not used Aiko, so I don't know whether it abides by this clause.
